src/features/subReddits/subredditInfo.js

```diff
@@ -2,105 +2,103 @@ import React, { useEffect } from "react";
 import { useSelector, useDispatch } from "react-redux";
 import {
   Card,
-  CardContent,
   ListSubheader,
   Typography,
   CardMedia,
-  ListItemIcon,
   ListItemText,
-  ListItem
+  ListItem,
 } from "@mui/material";
-import CakeIcon from "@mui/icons-material/Cake";
-import { Box } from "@mui/system";
 import { getSubredditInfo } from "./subRedditsSlice";
-import { numFormatter } from "../../components/helperFunctions";
+import { numFormatter } from "../../helpers/helperFunctions";
+import Loading from "../../components/loading";
+import { styled } from "@mui/material/styles";
+import { makeDate } from "../../helpers/helperFunctions";
+import AccountsInfo from "./accountsInfo";
+
+const AdvertiserCategory = styled("p")(({ theme }) => ({
+  color: theme.palette.primary.main,
+  lineHeight: "1.5rem",
+  border: "0.1rem solid rgba(255, 67, 0, 0.5)",
+  borderRadius: "5rem",
+  fontSize: "0.7rem",
+  textTransform: "none",
+  padding: theme.spacing(0, 0.5),
```

Comment:
hardcoded style value should be standardized and put into theme, although you use the theme, as a base, still hardcoded